dspace-server-webapp/src/test/java/org/dspace/app/rest/csv/CsvImportIT.java

```diff
@@ -222,22 +240,101 @@ private Item validateSpecificItemRelationCreationCsvImportMultiple(Collection co
     }
 
     private void performImportScript(String[] csv) throws Exception {
-        String filename = "test.csv";
-        BufferedWriter out = new BufferedWriter(
-            new OutputStreamWriter(
-                new FileOutputStream(filename), "UTF-8"));
-        for (String csvLine : csv) {
-            out.write(csvLine + "\n");
+        InputStream inputStream = new ByteArrayInputStream(String.join(System.lineSeparator(),
+                                                                       Arrays.asList(csv))
+                                                                 .getBytes(StandardCharsets.UTF_8));
+
+        MockMultipartFile bitstreamFile = new MockMultipartFile("file",
+                                                                "test.csv", MediaType.TEXT_PLAIN_VALUE,
+                                                                inputStream);
+
+        AtomicReference<Integer> idRef = new AtomicReference<>();
+
+        LinkedList<DSpaceCommandLineParameter> parameters = new LinkedList<>();
+        parameters.add(new DSpaceCommandLineParameter("-f", "test.csv"));
+        parameters.add(new DSpaceCommandLineParameter("-s", ""));
+
+        List<ParameterValueRest> list = parameters.stream()
+                                                  .map(dSpaceCommandLineParameter -> dSpaceRunnableParameterConverter
+                                                      .convert(dSpaceCommandLineParameter, Projection.DEFAULT))
+                                                  .collect(Collectors.toList());
+
+        try {
+            String token = getAuthToken(admin.getEmail(), password);
+
+            getClient(token)
+                .perform(fileUpload("/api/system/scripts/metadata-import/processes").file(bitstreamFile)
+                                                                                    .param("properties",
+                                                                                           new Gson().toJson(list)))
+                .andExpect(status().isAccepted())
+                .andDo(result -> idRef
+                    .set(read(result.getResponse().getContentAsString(), "$.processId")));
+            String t = "";
+        } finally {
+            ProcessBuilder.deleteProcess(idRef.get());
         }
-        out.flush();
-        out.close();
-        out = null;
+    }
+
+    @Test
+    public void createRelationshipsWithCsvImportWithSpecifiedEPersonParameterTest() throws Exception {
```

Comment:
We may want to move this test to `dspace-api`, as it looks like it's testing the *command-line* script can still use `-e`.  It's a good test, but it doesn't belong in the `dspace-server-webapp`, as the `dspace-server-webapp` is NOT allowed to use `-e`.  So, I think we should have a test in this class that proves the REST endpoint **cannot** use `-e` but still succeeds without it.

Reply:
@tim-atmire There might be a bit confusion about what this test does. This test checks that if you provide the "-e" option to the metadata-import script from the REST API that the script rejects this.

To make it a bit more clear, we made the following alterations:
* We renamed the method
* We don't just check that the item wasn't imported, but also check that the process has failed due to the invalid option.